ledger/store/src/block/mod.rs

```diff
@@ -1147,6 +1152,11 @@ impl<N: Network, B: BlockStorage<N>> BlockStore<N, B> {
     pub fn dev(&self) -> Option<u16> {
         self.storage.dev()
     }
+
+    /// Either enables or disables the atomic override.
+    pub fn flip_atomic_override(&self) -> Result<bool> {
+        self.storage.flip_atomic_override()
```

Comment:
Why are you only flipping one map?

Reply:
same as https://github.com/AleoHQ/snarkVM/pull/2296#discussion_r1460329080